mmv1/api/resource.go

```diff
@@ -220,6 +220,9 @@ type Resource struct {
 	// corresponding OiCS walkthroughs.
 	Examples []resource.Examples
 
+	// Samples for generating tests and documentation
+	Samples []resource.Sample
```

Comment:
```suggestion
	Samples []*resource.Sample
```

This might need to be a pointer (ran into a problem if it's not on my end.